jupyter_server/extension/utils.py

```diff
@@ -0,0 +1,70 @@
+import importlib
```

Comment:
Didn't we say we were going to pull this file out and make an issue?

Reply:
Good question! I've stripped out most of the unused logic from this file. 

I'm *still* using this submodule for helpful functions and custom exception classes, though.

Reply:
Cool